internal/producer/producer.go

```diff
@@ -0,0 +1,53 @@
+package producer
+
+import (
+	"github.com/Shopify/sarama"
+)
+
+// Producer is an interface for sending data to kafka
+type Producer interface {
+	Push(topic string, message []byte) error
+	Close()
+}
+
+// NewProducer returns Producer
+func NewProducer(brokersURL []string) Producer {
+	config := sarama.NewConfig()
+	config.Producer.Return.Successes = true
+	config.Producer.RequiredAcks = sarama.WaitForAll
+	config.Producer.Retry.Max = 5
+	conn, err := sarama.NewSyncProducer(brokersURL, config)
+	if err != nil {
+		panic("could not initialize producer")
+	}
+	return &producer{
+		conn: conn,
+	}
+}
+
+// producer is a Producer implementation
+type producer struct {
+	conn sarama.SyncProducer
+}
+
+// Push sends byte message to kafka
+func (p *producer) Push(topic string, message []byte) error {
+	msg := &sarama.ProducerMessage{
+		Topic: topic,
+		Value: sarama.StringEncoder(message),
+	}
+
+	_, _, err := p.conn.SendMessage(msg)
+	if err != nil {
+		return err
```

Comment:
Рекомендую использовать оборачивание ошибки.
Здесь нет стек трейса, как было бы в случае с эксепшенами в php или java.
Сейчас ошибка возвращается наверх и на вызывающей стороне будет не очень ясно как эта ошибка получилась.

Поэтому в go принят подход, когда ошибку "враппят".
Есть хороший пакет https://github.com/pkg/errors#readme
В самой гошке тоже добавили такой механизм через вызов:
```
return fmt.Errorf("example description: %w", err)
```
Вот подробности https://go.dev/blog/go1.13-errors

```suggestion
		return fmt.Errorf("cannot SendMessage: %w", err)
```

Не забудь импорт `fmt`.